addons/l10n_eu_service/tests/test_oss.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
+
+from odoo.addons.l10n_eu_service.models.eu_tag_map import EU_TAG_MAP
+from odoo.addons.account.tests.account_test_savepoint import AccountTestInvoicingCommon
+from odoo.tests import tagged
+
+
+@tagged('post_install_l10n', '-at_install')
+class TestOSSBelgium(AccountTestInvoicingCommon):
+
+    @classmethod
+    def setUpClass(self, chart_template_ref='l10n_be.l10nbe_chart_template'):
+        try:
+            super().setUpClass(chart_template_ref=chart_template_ref)
+        except ValueError as e:
+            if e.args[0] == "External ID not found in the system: l10n_be.l10nbe_chart_template":
+                self.skipTest(self, reason="Belgian CoA is required for this testSuite but l10n_be isn't installed")
+            else:
+                raise e
+        self.company_data['company'].country_id = self.env.ref('base.be')
+        self.company_data['company']._map_eu_taxes()
```

Comment:
To get rid of this, I would need to put something like this:
```py
cls.company_data = cls.setup_company_data('company_1_data', chart_template, country_id=cls.env.ref('base.be').id)
```
here:
https://github.com/odoo/odoo/blob/d1413a2f06bcb44b1fbb6dba8658e698c8da4018/addons/account/tests/account_test_savepoint.py#L50-L55

I don't have any other way of doing that as the environment isn't available before calling `super().setUpClass(chart_template_ref=chart_template_ref)` and setting an arbitrary country for all the other working test depending on this class is probably not acceptable.

The only alternative I have would be to call `setup_company_data` in the setup myself but that would mean having again a new company created with the whole chart of account setup, lowering the perf of this test.

The current solution is just the same as the user "finally" setting the country on his company and clicking on the button "refresh eu tax mapping"
https://github.com/odoo/odoo/blob/0e7c289003a0a69075188dbfaea724919dac32a4/addons/l10n_eu_service/models/res_config_settings.py#L10-L11